mautic/mautic
# Outdated packages updated. Only minor versions.

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Not really
| New feature? | Not really
| Automated tests included? | /
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

I used `$ composer outdated -D` command to find out which dependencies are outdated and can be updated without any harm.

Update of the CS Fixer probably added new rule to remove comments with duplicated information. So I run the CS Fixer and that's the big commit.
 
#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Click around. There should not be any error.
